src/main/resources/org/sonar/plugins/roslynsdk/sqale.xml

```diff
@@ -1,20 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<sqale>
-  <chc>
-    <chc>
-      <key>INSTRUCTION_RELIABILITY</key>
-      <chc>
-        <rule-key>S1000</rule-key>
-        <prop>
-          <key>remediationFunction</key>
-          <txt>CONSTANT_ISSUE</txt>
-        </prop>
-        <prop>
-          <key>offset</key>
-          <txt />
-          <val>15min</val>
-        </prop>
-      </chc>
-    </chc>
-  </chc>
-</sqale>
```

Comment:
I've added a test in the C# repo to check that the generated plugin does not contain a sqale file.